### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by KellyMerrick**

### CODE REVIEW

**Review by timhuynh94 — APPROVED** [review]

**Review by plyr4 — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: clone
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
```
Comment (wass3rw3rk) [inline .github/workflows/build.yml:15]: can move to use exact versions, like we do on core repos? example: https://github.com/go-vela/server/blob/main/.github/workflows/build.yml#L16

File: .golangci.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # outputs it results from the linters it executes.
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: colored-line-number
+  formats: colored-line-number
```
Comment (wass3rw3rk) [inline .golangci.yml:11]: is this valid? i would expect something more along the lines of https://golangci-lint.run/usage/configuration/#output-configuration or https://github.com/go-vela/server/blob/9bb4acca095dc88f92c48dd45f9bdd587740115b/.golangci.yml#L11-L14

**Review by wass3rw3rk — CHANGES_REQUESTED** [review]

File: .golangci.yml (hunk 2 above)
Comment (KellyMerrick) [reply .golangci.yml:11]: This was modified based on:

$ golangci-lint run
WARN [config_reader] The configuration option `output.format` is deprecated, please use `output.formats`

File: .github/workflows/build.yml (hunk 1 above)
Comment (KellyMerrick) [reply .github/workflows/build.yml]: updated

File: .golangci.yml (hunk 2 above)
Comment (wass3rw3rk) [reply .golangci.yml:11]: right, bit looks like it expects an array of objects, not a string?

Comment (KellyMerrick) [reply .golangci.yml]: corrected

**Review by wass3rw3rk — APPROVED** [review]

**Review by timhuynh94 — APPROVED** [review]

**Pull request merged**